ethernet_init/Kconfig.projbuild

```diff
@@ -0,0 +1,251 @@
+            range ENV_GPIO_RANGE_MIN ENV_GPIO_OUT_RANGE_MAX
+            default 32 if IDF_TARGET_ESP32
+            default 7 if IDF_TARGET_ESP32S2 || IDF_TARGET_ESP32S3
+            default 8 if IDF_TARGET_ESP32C3
+            default 21 if IDF_TARGET_ESP32C6
+            default 3 if IDF_TARGET_ESP32C2
+            default 11 if IDF_TARGET_ESP32H2
+            help
+                Set the GPIO number used by SPI CS1, i.e. Chip Select associated with the second SPI Eth module.
+
+        config EXAMPLE_ETH_SPI_INT0_GPIO
+            int "Interrupt GPIO number SPI Ethernet module #1"
+            range ENV_GPIO_RANGE_MIN ENV_GPIO_IN_RANGE_MAX
+            default 4 if IDF_TARGET_ESP32 || IDF_TARGET_ESP32S2 || IDF_TARGET_ESP32C3 || IDF_TARGET_ESP32S3
+            default 4 if IDF_TARGET_ESP32C2 || IDF_TARGET_ESP32C6
+            default 10 if IDF_TARGET_ESP32H2
+            help
+                Set the GPIO number used by the first SPI Ethernet module interrupt line.
+
+        config EXAMPLE_ETH_SPI_INT1_GPIO
+            depends on EXAMPLE_SPI_ETHERNETS_NUM > 1
+            int "Interrupt GPIO number SPI Ethernet module #2"
+            range ENV_GPIO_RANGE_MIN ENV_GPIO_IN_RANGE_MAX
+            default 33 if IDF_TARGET_ESP32
+            default 5 if IDF_TARGET_ESP32S2 || IDF_TARGET_ESP32C3 || IDF_TARGET_ESP32S3 || IDF_TARGET_ESP32C2
+            default 5 if IDF_TARGET_ESP32C6
+            default 9 if IDF_TARGET_ESP32H2
+            help
+                Set the GPIO number used by the second SPI Ethernet module interrupt line.
+
+        config EXAMPLE_ETH_SPI_PHY_RST0_GPIO
+            int "PHY Reset GPIO number of SPI Ethernet Module #1"
+            range -1 ENV_GPIO_OUT_RANGE_MAX
+            default -1
+            help
+                Set the GPIO number used to reset PHY chip on the first SPI Ethernet module.
+                Set to -1 to disable PHY chip hardware reset.
+
+        config EXAMPLE_ETH_SPI_PHY_RST1_GPIO
+            depends on EXAMPLE_SPI_ETHERNETS_NUM > 1
+            int "PHY Reset GPIO number of SPI Ethernet Module #2"
+            range -1 ENV_GPIO_OUT_RANGE_MAX
+            default -1
+            help
+                Set the GPIO number used to reset PHY chip on the second SPI Ethernet module.
+                Set to -1 to disable PHY chip hardware reset.
+
+        config EXAMPLE_ETH_SPI_PHY_ADDR0
+            int "PHY Address of SPI Ethernet Module #1"
+            range 0 31
+            default 1
+            help
+                Set the first SPI Ethernet module PHY address according your board schematic.
+
+        config EXAMPLE_ETH_SPI_PHY_ADDR1
+            depends on EXAMPLE_SPI_ETHERNETS_NUM > 1
+            int "PHY Address of SPI Ethernet Module #2"
+            range 0 31
+            default 1
+            help
+                Set the second SPI Ethernet module PHY address according your board schematic.
+    endif # EXAMPLE_USE_SPI_ETHERNET
+endmenu
```

Comment:
Could you please add support for CONFIG_ETH_USE_OPENETH, so that it's easier to test the applications where `ethernet_init` is used in QEMU?

Reply:
I've created a jira task for adding CONFIG_ETH_USE_OPENETH support.
https://jira.espressif.com:8443/browse/IDF-8238